src/switch_core_media.c

```diff
@@ -6079,6 +6079,35 @@ SWITCH_DECLARE(uint8_t) switch_core_media_negotiate_sdp(switch_core_session_t *s
 					switch_channel_set_variable(session->channel, "rtp_group", attr->a_value);
 				} else if (!strcasecmp(attr->a_name, "fingerprint") && !zstr(attr->a_value)) {
 					got_crypto = 1;
+				} else if (!strcasecmp(attr->a_name, "extmap") && attr->a_value) {
+					char *tmp = switch_mprintf("%s", attr->a_value);
+					char *argv[4] = { 0 };
+					int argc;
+
+					argc = switch_separate_string(tmp, ' ', argv, (sizeof(argv) / sizeof(argv[0])));
+					switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Checking extmap header %s\n", attr->a_value);
+
+					if (argc > 0 && !zstr(argv[1])) {
+						if (!strcasecmp(argv[1], SWITCH_MEDIA_EXTENSIONS_AUDIO_LEVEL)) {
+							if ((val = switch_channel_get_variable(session->channel, "force_rtp_ext_audio_level_events_negotiation")) && switch_true(val)) {
```

Comment:
It should be prefix with `rtp` so I would suggest to rename this to `rtp_force_ext_audio_level_events_negotiation`

Reply:
And with telnyx, so maybe `telnyx_rtp_force_ext_audio_level_events_negotiation`